library/Customizer/Sections/Footer.php

```diff
@@ -16,41 +16,23 @@ public function __construct($panelID)
         ));
 
         \Kirki::add_field(\Municipio\Customizer::KIRKI_CONFIG, [
-            'type'        => 'background',
-            'settings'    => 'footer_background',
-            'label'       => esc_html__('Footer background', 'municipio'),
-            'description' => esc_html__('Set the background of the footer area.', 'municipio'),
+            'type'        => 'slider',
+            'settings'    => 'footer_logotype_height',
+            'label'       => esc_html__('Footer logotype height', 'municipio'),
             'section'     => self::SECTION_ID,
-            'default'     => [
-                'background-color'      => 'rgba(20,20,20,.8)',
-                'background-image'      => '',
-                'background-repeat'     => 'repeat',
-                'background-position'   => 'center center',
-                'background-size'       => 'contain',
-                'background-attachment' => 'scroll',
+            'transport' => 'auto',
+            'default'     => 6,
+            'choices'     => [
+                'min'  => 3,
+                'max'  => 12,
+                'step' => 1,
+            ],
+            'output' => [
+                [
+                    'element'   => ':root',
+                    'property'  => '--c-footer-logotype-height-option',
```

Comment:
the option suffix seems redundant and we don't appear to use it elsewhere
```suggestion
                    'property'  => '--c-footer-logotype-height',
```